LINKS.md

```diff
@@ -2,20 +2,23 @@
 * [500px](https://500px.com/creativecommons) - Banco de imagens gratuitas
 * [Burst](https://pt.shopify.com/burst) - Plataforma de imagens do serviço de ecommerce Shopify
 * [Cupcake](http://cupcake.nilssonlee.se/) - Imagens gratuitas para uso comercial
+* [Diversidade](https://github.com/JulianaHelena5/BancoDeImagensComDiversidade) - Bancos de Imagens com Pessoas Diversas, maior acesso e representatividade.
```

Comment:
Remover ponto final, ordenar alfabeticamente e diminuir tamanho de descrição do link. Mais informações sobre padronização do projeto, acessar o [guia de contribuição](https://github.com/gabrielcmarinho/links-uteis/blob/master/CONTRIBUTING.md).

Reply:
E tomar cuidado com letras maiúsculas, tendo uso restrito para substantivos próprios, siglas, símbolos, nomes de instituições e início de frases.